### ISSUE TITLE
Search and Filtering

### ISSUE THREAD

**Issue by NtsikaMngoma** [jhipster/jhipster-dotnetcore#360]
Is there a way to get a Jhipster Repo with NEST implemented, with elasticsearch like how there is one with the Spring version. If not what other ways can the existing repos in dotnet apply searching and filtering using multiple query params for the server and client?

**Comment by NtsikaMngoma** [comment]
On the client application, when I type on the search bar to search, **Not Found** is returned, I assume this is based around NEST not setup on the API?

**Comment by nicolas63** [comment]
Something like main generator (https://www.jhipster.tech/entities-filtering/) ?

**Comment by NtsikaMngoma** [comment]
Sort of, I just need something that can search a filter with multiple params, from username, firstname, surname, etc. Elasticsearch/solr could be suitable for data searching.

**Comment by NtsikaMngoma** [comment]
> Something like main generator (https://www.jhipster.tech/entities-filtering/) ?

Yeah this could also work, but for dotnet, just to filter with the GET request, then autocomplete on the client if possible.

**Comment by vraravam** [comment]
I have just released a linq-based filter library that was modeled on the java-based jHipster: https://www.nuget.org/packages/Linq.Query.Filters/ - please see if this is useful. I plan to try and work on weekends to learn contributing to jHipster-dotnetcore and see if I can raise a PR for generating this OOB.
